packages/legacy-client/src/client/objects/convertWireToOsdkObject.ts

```diff
@@ -68,7 +69,7 @@ function createPrototype<
 
   // add the relevant keys for the link types associated with this object type
   for (
-    const [k, { multiplicity, targetType }] of Object.entries(objDef.links)
+    const [k, { multiplicity }] of Object.entries(objDef.links)
```

Comment:
Can we stop serializing `targetType` entirely into the OntologyDefinition? I guess we still use it for type inference?

Reply:
Yeah I think we'd still need it for type inference, and also just readability of the definition